### PR TITLE
app-backup/rsnapshot: some fixes

### PR DESCRIPTION
**Pull request by bor**

### CODE REVIEW

**Comment by sbraz** [comment]
I'd use sed instead of a patch since the changes are trivial. And you won't have to change it at each update.

**Comment by bor** [comment]
Good idea, thanks again.

File: app-backup/rsnapshot/rsnapshot-1.4.2.ebuild [hunk 1]
```diff
@@ -20,7 +20,15 @@ RDEPEND=">=dev-lang/perl-5.8.2
 		>=net-misc/rsync-2.6.0"
 DEPEND="${RDEPEND}"
 
+src_prepare() {
+	# remove '/etc/' since we don't place it here, bug #461554
+	sed -i -e 's:/etc/rsnapshot.conf.default:rsnapshot.conf.default:' rsnapshot-program.pl || die
+	eapply_user
```
Comment (SoapGentoo) [inline app-backup/rsnapshot/rsnapshot-1.4.2.ebuild]: please call `default` instead of `eapply_user`

Comment (bor) [inline app-backup/rsnapshot/rsnapshot-1.4.2.ebuild]: fixed, thanks for the point

**Comment by bor** [comment]
@monsieurp I call you to help with merge :)

**Pull request closed without merging**